scripts/db/schema.sql

```diff
@@ -45,6 +72,105 @@ CREATE TYPE enum_assets_type AS ENUM (
 
 --
 
+CREATE TYPE enum_users_canvas_enrollment_state AS ENUM (
+    'active',
+    'completed',
+    'inactive',
+    'invited',
+    'rejected'
+);
+
+--
+
+CREATE TABLE activities (
+    id integer NOT NULL,
+    type enum_activities_type NOT NULL,
+    object_id integer,
+    object_type enum_activities_object_type NOT NULL,
+    metadata json,
+    asset_id integer,
+    course_id integer NOT NULL,
+    user_id integer NOT NULL,
+    actor_id integer,
+    reciprocal_id integer,
+    created_at timestamp with time zone NOT NULL,
+    updated_at timestamp with time zone NOT NULL
+);
```

Comment:
The "activities" table hangs around for engagement index purposes. The "events" table will not be implemented.